conf/messages.en

```diff
@@ -48,6 +48,7 @@ date.month.decimal.error = Month must not include decimals
 date.month.outOfRange.error = Month must be a number between 1 and 12
 date.month.leadingBlankSpace.error = Month must not have leading spaces
 date.month.trailingBlankSpace.error = Month must not have trailing spaces
+date.decimal.error = Date must be a real date
```

Comment:
Does this need changing/renaming from `date.decimal.error` as we're removing decimal specific error messages? If it's been left like this for easier mental processing/readability then ignore no worries :+1:

Reply:
We will revisit this once we get the actual error message.